src/components/UserCard/index.jsx

```diff
@@ -59,34 +110,123 @@ export default function User(props) {
-                  returnOptionIfExists(details.options, 'actions', []),
-                  returnOptionIfExists(details.options, 'products', [])
+      {/* We don't need to check roles here because users without permissions
+            cannot hold roles */}
+      {Object.keys(permissions).length > 0 && (
+        <CardContent>
+          <List>
+            {Object.entries(permissions).map(([permission, details]) => (
+              <ListItem key={permission}>
+                <ListItemIcon>
+                  <KeyVariantIcon />
+                </ListItemIcon>
+                <ListItemText
+                  primary={
+                    <Fragment>
+                      {getPermissionString(
+                        permission,
+                        returnOptionIfExists(details.options, 'actions', []),
+                        returnOptionIfExists(details.options, 'products', [])
+                      )}
+                      {scheduledPermissions[permission] && (
+                        <span className={classes.propertyWithScheduledChange} />
+                      )}
+                    </Fragment>
+                  }
+                />
+              </ListItem>
+            ))}
+            {Object.keys(roles).length > 0 && (
+              <ListItem>
+                <ListItemIcon>
+                  <AccountGroupIcon />
+                </ListItemIcon>
+                <ListItemText>
+                  holds the {getRolesString(Object.keys(roles))}
+                </ListItemText>
+              </ListItem>
+            )}
+          </List>
+        </CardContent>
+      )}
+      {Object.keys(scheduledPermissions).length > 0 &&
+        Object.entries(scheduledPermissions).map(([permission, details]) => (
+          <Fragment key={permission}>
+            {Object.keys(permissions).length > 0 && <Divider />}
+            <CardContent className={classes.scheduled}>
+              <div>
+                <StatusLabel
+                  className={classes.statusLabel}
+                  state={getStatus(details.change_type)}
+                />
+              </div>
+              <Grid container>
+                {permissions[permission] && (
+                  <Fragment>
+                    <Grid
+                      item
+                      xs={5}
+                      className={classes.scheduledChangeDescriptionMarginLeft}>
+                      {getPermissionString(
+                        permission,
+                        returnOptionIfExists(
+                          permissions[permission].options,
+                          'actions',
+                          []
+                        ),
+                        returnOptionIfExists(
+                          permissions[permission].options,
+                          'products',
+                          []
+                        )
+                      )}
+                    </Grid>
+                    <Grid item xs={1}>
```

Comment:
Center the arrow in the grid. I think you could do `display: 'flex', justifyContent: 'center'` on the Grid.